core/src/apps/bitcoin/sign_tx/matchcheck.py

```diff
@@ -107,6 +107,18 @@ def attribute_from_tx(self, txio: TxInput | TxOutput) -> Any:
         return multisig.multisig_fingerprint(txio.multisig)
 
 
+class PubkeysOrderChecker(MatchChecker):
+    def attribute_from_tx(self, txio: TxInput | TxOutput) -> Any:
+        if not txio.multisig:
+            return None
+        return txio.multisig.pubkeys_order
```

Comment:
I think it's perfectly clean to hash the `pubkeys_order` attribute into the multisig fingerprint, since the fingerprint already contains several attributes of `MultisigRedeemScriptType` (XPUBs and threshold). Then we can get rid of `PubkeysOrderChecker` while maintaining the same behavior.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4351/commits/4e52009e889fca6cc7f8f2905c9b3ea2250ea429.